### ISSUE TITLE
install errors - potential conflict with tensorflow

### ISSUE THREAD

**Issue by kangsan422** [theislab/campa#11]
Dear Developers,

I am trying to install campa on my Apple M1 Max machine;
Please note that I get the same error regardless of whether tensorflow is installed in my conda environment.
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/72509917/178323546-e34491b9-d6b0-422e-9ff0-45852ef77c17.png">

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/72509917/178323467-205463b8-c708-4969-80f0-b27a3b8c9990.png">

Normal installation of tensorflow does not work in my machine, so I had to do following to install tensorflow: 
conda install -c apple tensorflow-deps==2.7.0
python -m pip install tensorflow-macos==2.7.0
python -m pip install tensorflow-metal

After installing tensorflow, I can see below:
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/72509917/178324002-0e8af675-0abf-452e-9f42-b3c28c2c0282.png">

More details of pip freeze;

absl-py==1.1.0
appnope==0.1.3
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.5
astunparse==1.6.3
attrs==21.4.0
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.0
brotlipy @ file:///Users/runner/miniforge3/conda-bld/brotlipy_1648854242877/work
cached-property @ file:///home/conda/feedstock_root/build_artifacts/cached_property_1615209429212/work
cachetools==5.2.0
certifi==2022.6.15
cffi @ file:///Users/runner/miniforge3/conda-bld/cffi_1636046173594/work
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1644853463426/work
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1602866480661/work
conda-package-handling @ file:///Users/runner/miniforge3/conda-bld/conda-package-handling_1649385125392/work
cryptography @ file:///Users/runner/miniforge3/conda-bld/cryptography_1652967108255/work
cycler==0.11.0
czifile==2019.7.2
debugpy==1.6.0
decorator==5.1.1
defusedxml==0.7.1
entrypoints==0.4
executing==0.8.3
fastjsonschema==2.15.3
flatbuffers==1.12
fonttools==4.33.3
gast==0.4.0
google-auth==2.8.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio @ file:///Users/runner/miniforge3/conda-bld/grpc-split_1655728648515/work
h5py @ file:///Users/runner/miniforge3/conda-bld/h5py_1637964045648/work
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1642433548627/work
imageio==2.19.3
importlib-metadata==4.11.4
ipykernel==6.15.0
ipython==8.4.0
ipython-genutils==0.2.0
ipywidgets==7.7.1
jedi==0.18.1
Jinja2==3.1.2
joblib @ file:///home/conda/feedstock_root/build_artifacts/joblib_1633637554808/work
jsonschema==4.6.0
jupyter==1.0.0
jupyter-client==7.3.4
jupyter-console==6.4.4
jupyter-core==4.10.0
jupyterlab-pygments==0.2.2
jupyterlab-widgets==1.1.1
keras==2.9.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.3
libclang==14.0.1
libmambapy @ file:///Users/runner/miniforge3/conda-bld/mamba-split_1649138467459/work/libmambapy
llvmlite==0.38.1
Markdown==3.3.7
MarkupSafe==2.1.1
matplotlib==3.5.2
matplotlib-inline==0.1.3
mistune==0.8.4
nbclient==0.6.4
nbconvert==6.5.0
nbformat==5.4.0
nest-asyncio==1.5.5
networkx==2.8.4
notebook==6.4.12
numba==0.55.2
numpy @ file:///Users/runner/miniforge3/conda-bld/numpy_1653325964689/work
oauthlib==3.2.0
opencv-python==4.6.0.66
opt-einsum==3.3.0
packaging==21.3
pandas==1.4.2
pandocfilters==1.5.0
parso==0.8.3
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.1.1
prometheus-client==0.14.1
prompt-toolkit==3.0.29
protobuf==3.19.4
psutil==5.9.1
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycosat @ file:///Users/runner/miniforge3/conda-bld/pycosat_1649384941891/work
pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1636257122734/work
Pygments==2.12.0
pynndescent==0.5.7
pyOpenSSL @ file:///home/conda/feedstock_root/build_artifacts/pyopenssl_1643496850550/work
pyparsing==3.0.9
pyrsistent==0.18.1
PySocks @ file:///Users/runner/miniforge3/conda-bld/pysocks_1648857374584/work
python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/python-dateutil_1626286286081/work
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1647961439546/work
PyWavelets==1.3.0
pyzmq==23.2.0
qtconsole==5.3.1
QtPy==2.1.0
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1641580202195/work
requests-oauthlib==1.3.1
rsa==4.8
ruamel-yaml-conda @ file:///Users/runner/miniforge3/conda-bld/ruamel_yaml_1653464548430/work
scikit-image==0.19.3
scikit-learn @ file:///Users/runner/miniforge3/conda-bld/scikit-learn_1652976950430/work
scipy @ file:///Users/runner/miniforge3/conda-bld/scipy_1653074075583/work
seaborn==0.11.2
Send2Trash==1.8.0
six @ file:///home/conda/feedstock_root/build_artifacts/six_1620240208055/work
soupsieve==2.3.2.post1
stack-data==0.3.0
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow-estimator==2.9.0
tensorflow-macos==2.9.2
tensorflow-metal==0.5.0
termcolor==1.1.0
terminado==0.15.0
threadpoolctl @ file:///home/conda/feedstock_root/build_artifacts/threadpoolctl_1643647933166/work
tifffile==2022.5.4
tinycss2==1.1.1
tornado==6.1
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1649051611147/work
traitlets==5.3.0
typing_extensions==4.2.0
umap-learn==0.5.3
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1647489083693/work
wcwidth==0.2.5
webencodings==0.5.1
Werkzeug==2.1.2
widgetsnbextension==3.6.1
wrapt==1.14.1
zipp==3.8.0
